app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```

Comment:
- [ ] Just like the comment model, here you need to create a new method that updates the likes counter for a post after you add a new like, so please implement that!